cli/container.go

```diff
@@ -110,6 +118,7 @@ func createContainer(options runvOptions, vm *hypervisor.Vm, container, bundle,
 			Pid:     shim.Pid,
 			Bundle:  bundle,
 		},
+		stat.StartTime, 0,
```

Comment:
golang does not allow mixing field:value and value initializers. e.g.,
`cli/container.go:114: mixture of field:value and value initializers`